doc/user/layouts/shortcodes/cloud-connection-details.html

```diff
@@ -1,8 +1,20 @@
-1. Click the deployment card, then **download** and **unzip** the certificate package.
+1. Click on a deployment to see the individual deployment view.
+
+1. If you are connecting through Tailscale VPN, go to the Integrations card, then the Tailscale tab. Click **Enable**, then enter the Tailscale auth key in the Tailscale Authentication dialog and click **Save**.
+
+    For help generating an auth key, please see the [Tailscale documentation](https://tailscale.com/kb/1085/auth-keys/).
+
+    **Note:** Tailscale integration must enabled separately for each deployment.
+
+1. In the Connect card, go to the psql tab and click **Download certificates**.
+
+1. Unzip the certificate package.
+
+    The certificate package is named `<deployment-name>.zip` and contains the TLS certificates and keys you need to connect.
 
 1. To connect to the deployment, you can use any [Materialize-compatible CLI](/connect/cli), like `psql` or `mzcli`. If you have `psql` installed on your machine, proceed to the next step.
 
-    Otherwise, follow the [installation instructions](/install/#cli-connections).
+    Otherwise, follow the [CLI installation instructions](/install/#cli-connections).
 
 1. Open a terminal window on your machine, copy the `psql` command from the connection dialog and run it from the directory that contains the certificates.
```

Comment:
We probably want to tell them to replace the hostname in the `psql` command with the address of their materialized in Tailscale if they are using Tailscale to connect.

I unfortunately don't know of a good way to get this address on our side, so we can't easily populate that for them.